pycbc/tmpltbank/bank_conversions.py

```diff
@@ -88,20 +83,56 @@ def get_bank_property(parameter, bank, template_ids,
     # These just give things already in the bank
     if parameter in bank:
         values = bank[parameter][:][template_ids]
-
-    # These things may be in the bank, but if not, we need to calculate
-    elif parameter in ['template_duration', 'duration']:
-        if 'template_duration' in bank:
+    # Duration may be in the bank, but if not, we need to calculate
+    elif parameter.endswith('duration'):
+        duration_approximant = bank['template_duration'][:][template_ids] \
+            if 'approximant' in bank \
+            else np.repeat("SPAtmplt", len(template_ids))
```

Comment:
This is effectively changing the default approximant for duration calculation from SEOBNRv4 to SPAtmplt. Not sure this is necessarily a good idea, and if it is, a plotting PR is probably not the place to do it. Can you keep the `duration_approximant` as a kwarg with the current default?

Reply:
Also, I guess line 88 meant to use `approximant` as index, not `template_duration`?